localStorage.js

```diff
@@ -0,0 +1,17 @@
+const form = document.querySelector('.formSection');
+const {name, email, msg } = form.elements;
+if (!localStorage.getItem('formObjectData')) {
+  const formData = { fnameData: '', emailData: '', messageData: '' };
+  localStorage.setItem('formObjectData', JSON.stringify(formData));
+}
+const reservedata = (element, val) => {
+  element.addEventListener('change', () => {
+    const data = element.value;
+    const formData = JSON.parse(localStorage.getItem('formObjectData'));
+    formData[val] = data;
+    localStorage.setItem('formObjectData', JSON.stringify(formData));
+  });
```

Comment:
- [ ] You are successfully saving the form data to the localStorage. However, you are not retrieving it because when the page reloads, the data is lost. The contact form should be persistent to page reloads. Kindly fix this 🙏

Reply:
I reload the page and  do it many times and the information not lost 
![image](https://user-images.githubusercontent.com/34026019/188728522-6980093c-769a-4efa-bc2c-6036f53675da.png)

Reply:
@codepantha

Reply:
Hi @ManarOmari. Sorry I was not able to reply instantly 🙏 @ShahierNashaat thanks for clarifying 👍